template.yaml

```diff
@@ -857,6 +878,333 @@ Resources:
+            - 'arn:aws:dynamodb:eu-west-2:${AccountId}:table/${AuthEnvironment}-'
+            - AccountId: !FindInMap [ EnvironmentConfiguration, !Ref Environment, authAccountId ]
+              AuthEnvironment: !FindInMap [ EnvironmentConfiguration, !Ref Environment, authEnvironment ]
+          FRONTEND_BASE_URL: !Sub
+            - https://signin.${ServiceDomain}/
+            - ServiceDomain: !FindInMap [ EnvironmentConfiguration, !Ref Environment, serviceDomain ]
+          LOGIN_URI: !Sub
+            - https://signin.${ServiceDomain}/
+            - ServiceDomain: !FindInMap [ EnvironmentConfiguration, !Ref Environment, serviceDomain ]
+          ACCOUNT_INTERVENTION_SERVICE_ACTION_ENABLED: true
+          ACCOUNT_INTERVENTION_SERVICE_CALL_ENABLED: true
+          ACCOUNT_INTERVENTION_SERVICE_URI: !Sub
+            - '{{resolve:secretsmanager:${SecretArn}:SecretString}}'
+            - SecretArn: !Ref AccountInterventionsServiceUriSecret
+          ACCOUNT_INTERVENTION_SERVICE_ABORT_ON_ERROR: !If
+            - AisAbortOnError
+            - true
+            - false
+          IPV_AUDIENCE: !If
+            - IdentityEnabled
+            - !Sub
+              - https://identity.${ServiceDomain}/
+              - ServiceDomain: !FindInMap [ EnvironmentConfiguration, !Ref Environment, serviceDomain ]
+            - !Ref AWS::NoValue
+          IPV_AUTHORISATION_CALLBACK_URI: !If
+            - IdentityEnabled
+            - !Sub
+              - https://oidc.${ServiceDomain}/ipv-callback
+              - ServiceDomain: !FindInMap [ EnvironmentConfiguration, !Ref Environment, serviceDomain ]
+            - !Ref AWS::NoValue
+          IPV_AUTHORISATION_CLIENT_ID: !If
+            - IdentityEnabled
+            - authOrchestrator
+            - !Ref AWS::NoValue
+          IPV_AUTHORISATION_URI: !If
+            - IdentityEnabled
+            - !Sub
+              - https://identity.${ServiceDomain}/oauth2/authorize
+              - ServiceDomain: !FindInMap [ EnvironmentConfiguration, !Ref Environment, serviceDomain ]
+            - !Ref AWS::NoValue
+          IPV_BACKEND_URI: !If
+            - IdentityEnabled
+            - !Sub
+              - https://api.identity.${ServiceDomain}/
+              - ServiceDomain: !FindInMap [ EnvironmentConfiguration, !Ref Environment, serviceDomain ]
+            - !Ref AWS::NoValue
+          IPV_NO_SESSION_RESPONSE_ENABLED: true
+          IPV_TOKEN_SIGNING_KEY_ALIAS: !FindInMap [ EnvironmentConfiguration, !Ref Environment, ipvTokenAuthKeyArn ]
+          SPOT_QUEUE_URL: !If
+            - IdentityEnabled
+            - !Sub
+              - https://sqs.eu-west-2.amazonaws.com/${AccountId}/${AuthEnvironment}-spot-request-queue
+              - AccountId: !FindInMap [ EnvironmentConfiguration, !Ref Environment, authAccountId ]
+                AuthEnvironment: !FindInMap [ EnvironmentConfiguration, !Ref Environment, authEnvironment ]
+            - !Ref AWS::NoValue
```

Comment:
Just to check, are these `IdentityEnabled` cases correct in being based off of the condition (stage, integration or production)? just because the `IDENTITY_ENABLED` variable getting passed into the function is based off of whether it is build so don't quite align but I'm aware they're meant to not be quite the same

Reply:
It's a bit confusing, I'm going to add a comment. It's the difference between having the ipv api turned on (IDENTITY_ENABLED), and environments where identity actually works (IdentityEnabled). Sandpit has ipv api turned on for testing even though identity doesn't work